### ISSUE TITLE
Unable to pair with Bluetooth

### ISSUE THREAD

**Issue by kylegordon** [openDsh/dash#72]
# Issue

*Make sure you are running the latest version before reporting an issue.*

## Hardware

| Device             | Connection Method | Screen                       | Mobile Device     | Android Auto Version |
| ------------------ | ----------------- | ---------------------------- | ----------------- | -------------------- |
| Raspberry Pi 4 4GB | USB Cable         | Forwarded X Display | Umidigi F2 | v.1.1          |

Installation Method: Install.sh or Image

Fully updated Raspbian 10 Buster, installed from ./install.sh

## Description of problem:

When in the Bluetooth settings screen, 'Scan' successfully shows a list of devices. Clicking on the phone I wish to pair results in the following error at the command line

org.kde.bluez: PendingCall Error: "Protocol not available"

## Expected Result:

Some form of happy path user bluetooth pairing experience

**Comment by rsjudka** [comment]
so the bluetooth feature in dash - at least for now - will only connect, not pair

reason for that being is there's that whole authentication bit I just couldn't figure out how to do lol

**Comment by robert5974** [comment]
oh...I thought I had paired before.  It's always been this way?

**Comment by robert5974** [comment]
@kylegordon did you try to pair the device in the OS first?  That works for me and I believe was stated in the Readme.  At least at some point it was if it's not there now.  I would have to look.  After pairing using the OS, dash will connect easily.

**Comment by robert5974** [comment]
I am working with the Bluetooth service in Dash to see if I can inject pairing Bluetooth devices using the Bluez agent

**Comment by icecube45** [comment]
@robert5974 last I looked into it, the appropriate bindings weren't implemented for pairing - I'd be interested to learn more about what you're looking into, I have a couple of personal projects that could benefit from pairing support.

**Comment by robert5974** [comment]
Right, like @rsjudka said above, it only connects and does not pair so I have been researching and working towards implementing using the agent to authenticate device.  

I was going to begin with a "no pin" authentication for pairing to test functionality.  Adding the pin should be easy after proving that works.  To use a "No pin" authentication method is called the "NoInputNoOutput" option when using the Bluez agent.
